client/css/style.css

```diff
@@ -574,6 +574,10 @@ kbd {
 	color: #49505a;
 }
 
+#sidebar .close-tooltip {
+	float: right;
+}
+
```

Comment:
Tooltip gets bold on lobbies. I don't know if #1855 fixes it by accident.

Reply:
It's because the lobby name is bold, not sure of best fix

Reply:
Yeah I know, but I'm wondering if #1855 does not fix this. Their version and ours differ slightly and it might force to be non-bold.

Reply:
Never closed the loop, but yes, current primer-tooltip (updated after this PR was opened) fixed this.